python/py-pythonz/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        berdario pythonz 1.11.4
+github.tarball_from releases
+name                py-${github.project}
+
+master_sites        ${github.homepage}/archive/
+distname            ${github.project}-${github.version}
+worksrcdir          ${github.project}-${github.project}-${github.version}
+
+categories          python
```

Comment:
The `categories` line can be deleted because the python portgroup adds it already.